thentos-core/purescript/test/Main.purs

```diff
@@ -0,0 +1,6 @@
+module Test.Main where
+
+import Control.Monad.Eff.Console
+
+main = do
+  log "You should add some tests."
```

Comment:
I'll remove this? I'm all in favour of adding tests, of course, but right now the only thing we'd be testing is the backend.